docs/reference/module-api/module-options.md

```diff
@@ -126,6 +127,22 @@ module.exports = {
 }
 ```
 
+### `csrfExceptions`
```

Comment:
That... in other modules... is... a thing. Right. Documentation is good. Thanks for catching this.